spec/features/quotes/index_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'quotes index' do
+  describe 'as a visitor' do
+    describe 'when I search for a keyword from the landing page' do
+      it 'takes me to a new page' do
+        visit '/'
+
+        fill_in :query, with: "find"
+
+        click_on 'Search for Quotes'
+
+        expect(current_path).to eq('/quotes')
+      end
+
+      it 'gives the number of search results' do
+        visit '/'
+
+        fill_in :query, with: "find"
+        click_button 'Search for Quotes'
+        within '#total_results' do
+
+          expect(page).to have_content(51)
+        end
+      end
```

Comment:
since you've tested that your facade works, you could do something like:

```ruby
info = QuoteFacade.your_method(keyword)
info[:quotes].each do |quote|
   expect(quote).to have_content(quote.content)
   # ... and so on for each quote
end 
```